src/Microsoft.Data.Analysis/DataFrameColumn.BinaryOperations.cs

```diff
@@ -12,6 +12,9 @@ namespace Microsoft.Data.Analysis
 {
     public abstract partial class DataFrameColumn
     {
+        /// <summary>
+        /// Performs element wise addition
```

Comment:
`element-wise` or `elementwise`. It isn't 2 words, note how we capitalized the method names: `ElementwiseEquals`.